frontend/src/styles/globals.css

```diff
@@ -24,3 +24,15 @@
     @apply bg-background text-foreground;
   }
 }
+
+::-webkit-scrollbar {
+  width: 12px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px #65098E; 
```

Comment:
```suggestion
  box-shadow: inset 0 0 5px var(--primary); 
```

Reply:
I was using the css variables at first but for some reason the changes were not showing up on the site . Like no scrollbar was being visible .
<img width="960" alt="image" src="https://github.com/danielschuster-muc/potter-db/assets/100552235/26a92926-031c-449a-9c75-0c5a87bc4e4f">

Reply:
well then better use the #colors

Reply:
I think you have to wrap it with hsl, like this:
`hsl(var(--primary))`

Reply:
Yes , it's working now. Thanks for the suggestion.